language/move-lang/functional-tests/tests/diem/network_identity/network_identity.move

```diff
@@ -0,0 +1,88 @@
+/// Test to check that we can write identity into the NetworkIdentity and update it
```

Comment:
We are trying not to add new functional tests at the moment. Would it be possible to rewrite this as a unit test?
@tzakian can likely help

Reply:
Unit tests don't work for diem-framework.  And won't have support until later this half.

Reply:
Is the issue just for metadata items like DiemTimestamp? Otherwise these feel rather easy to describe in unit tests

Reply:
I talked to @tzakian directly previously, and he told me go forward with the functional tests for now.

Reply:
My guess is DiemTimstamp will be a blocker for this specific case, but we will  confer with Tim to be sure. I'd like to get a move on with test migration/cleanup, so would be nice if we could not add additional functional tests

Reply:
I also would like to be done with this PR, as I've been tossed around a bit on whether I should do one thing or the other.

Tim is currently on vacation, so I'd like to not bother him.  If it's a problem later, you can feel free to delete my tests.

Reply:
When @gregnazario was writing these tests we didn't support unit tests in the Diem Framework, so I suggested using the functional tests and I'll convert them over later. Even though we now support unit tests, I think it's fine to keep these tests as-is for now in this PR to avoid churn, and I'll convert these tests over once things land as part of the overall test conversions.

Reply:
Works for me! And I have no fucking clue whats going on with the nature of the rest of the PR lol. So I'd leave any other final review to @sblackshear or @tzakian or someone else equally more qualified :P